lco_ingester/fits.py

```diff
@@ -53,7 +53,7 @@ def remove_blacklist_headers(self):
     def normalize_null_values(self):
         #  Sometimes keywords use N/A to mean null
         for k, v in self.fits_dict.items():
-            if v == 'N/A' or v == 'UNKNOWN' or v == 'UNSPECIFIED':
+            if v == 'N/A' or v == 'UNKNOWN' or v == 'UNSPECIFIED' or v == 'NONE' or v == 'None':
```

Comment:
Will this catch the `'None   '` (with extra whitespace) case? Is the whitespace trimmed somewhere before this point?

Reply:
I thought we did a strip somewhere earlier, but didn't find it, so I added that in

Reply:
actually on second thought, we don't do a strip because the value isn't necessarily a string at this point. Instead I'll leave that portion as-is but switch the None and NONE check to be an `in` rather than `==` check.